src/components/HomeSectionClients/styled.ts

```diff
@@ -9,14 +9,31 @@ import fatLlamaUrl from './fat-llama.svg';
 import framerUrl from './framer.svg';
 import googleUrl from './google.svg';
 
-export const HomeLeftRightWrapper = styled(_HomeLeftRightWrapper)`
-  & + & {
-    margin-top: 36px;
-  }
+export const CasesItemsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
 `;
 
 const leftSpacing = 30;
 export const Content = styled.div`
+  flex-basis: 100%;
+  margin-bottom: 48px;
+  :nth-last-child(1) {
+    margin-bottom: 0px;
+  }
+  font-size: 16px;
+
+  ${media.notSmall`
+    flex-basis: 50%;
+    :nth-child(odd) {
+      padding-right: 72px;
```

Comment:
The layout looks great! However, a padding-right on the left column makes the left column narrower than the right one:

![CleanShot 2021-10-20 at 15 25 02@2x](https://user-images.githubusercontent.com/2953267/138092122-d9f765b7-c4b3-45ea-be80-6708e23b1ea0.png)

Could we make them equal width? (Perhaps a `gap` will also make more sense here. Or maybe not.)